Content/src/Client/Client.fs

```diff
@@ -70,34 +67,48 @@ module Server =
 
     open Shared
     open Fable.Remoting.Client
-    
+
     #if (deploy == "iis" && server != "suave")
     // normalize routes so that they work with IIS virtual path in production
     let normalizeRoutes typeName methodName =
         Route.builder typeName methodName
-        |> ServerPath.normalize   
+        |> ServerPath.normalize
 
     /// A proxy you can use to talk to server directly
     let api : ICounterApi =
       Remoting.createApi()
       |> Remoting.withRouteBuilder normalizeRoutes
       |> Remoting.buildProxy<ICounterApi>
-    #else 
+    #else
     /// A proxy you can use to talk to server directly
     let api : ICounterApi =
       Remoting.createApi()
       |> Remoting.withRouteBuilder Route.builder
       |> Remoting.buildProxy<ICounterApi>
-    #endif  
-
-
+    #endif
+#else
+// Fetch a data structure from specified url and using the decoder
+let fetchWithDecoder<'T> (url: string) (decoder: Decoder<'T>) (init: RequestProperties list) =
```

Comment:
wonder if this belongs in some lib!?

Reply:
@alfonsogarciacaro @MangelMaxime I took it from fable migration guide

Reply:
@forki Well, personally I rewrote my own version in my application because I need to handle refresh tokens etc.

In a previous issue https://github.com/MangelMaxime/Thoth/issues/130, I mentioned I could create a `Thoth.Fetch` package to host this kind of helpers. If you think it is needed, I can do it.

Reply:
I'm happy to consume `fetchAs` function from a library - indeed that seems like unnecessary boilerplate in application code

Reply:
I think it's better to keep Fable.Fetch and Thoth.Json packages independent and don't create a hard dependency between them just to enable a single function.

Reply:
The solution could be to create an extra package that has the dependency on both Fable.Fetch and Thoth.Json. Although I'm still in doubt if we should insist on Fable.Fetch or try something more idiomatic like Fable.SimpleHttp (which is also compatible with IE 11 without a polyfill).

Reply:
@alfonsogarciacaro I think @MangelMaxime proposed a third package which marries both

Reply:
Ups, I just noticed you were continuing the discussion below. If Maxime is already on it, that's fine :)

Reply:
so in other words: both packages stay independent. but if you want to use both anyway then you use the married thing and it will introduce the original packages as transitives

Reply:
Yes, I am working on Thoth.Fetch package which offers an API over Fable.Fetch + Thoth.Json.

Like that we still keep the independent low-level packages.